DEPLOYMENT.md

```diff
@@ -0,0 +1,43 @@
+# Deploying a release
+
+This document contains the steps required when releasing a new version of UMS.
+
+## Preparation:
+1.  Be on latest `master` branch
+1.  Pull translations from Crowdin and commit them
+1.  Update changelog with the changes since the last version, and the release date
+1.  Decide which kind of version bump is necessary. Bugfixes or refactoring should bump the third number, others should bump the second one.
+1.  Update the versions in the changelog and pom.xml (lines 33 and 66) including removing "-SNAPSHOT" from line 33
+1.  Create a tag of the new version (e.g. 6.7.2, not v6.7.2)
+1.  Push your commits and new tag
+
+## Build:
+1.  Run build.bat on Windows, which generates releases for Windows and Linux
+1.  Append version numbers to the files
```

Comment:
I wasn't thinking of the version number itself, but how to modify the filename - the file name pattern if you will.